examples/frameworks/nat_ag2_demo/src/nat_ag2_demo/ag2_research_team.py

```
@@ -0,0 +1,110 @@
+    user = ConversableAgent(
+        name="user", human_input_mode="NEVER"
+    )
+
+    pattern = AutoPattern(
+        initial_agent=researcher,
+        agents=[researcher, writer],
+        user_agent=user,
+        group_manager_args={"llm_config": llm_config},
+    )
+
+    result, ctx, last = initiate_group_chat(
```

Ruff (0.15.6)

[warning] Unpacked variable `ctx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)